kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -576,3 +620,17 @@ def send_always_counter(self, metric, scraper_config, hostname=None):
             # Determine the tags to send
             tags = self._metric_tags(metric.name, val, sample, scraper_config, hostname=custom_hostname)
             self.monotonic_count(metric_name_with_namespace, val, tags=tags, hostname=custom_hostname)
+
+    def _histogram_from_seconds_to_microseconds(self, metric, scraper_config):
```

Comment:
did you considered moving the function to the `OpenMetricsBaseCheck` class? like this the method can be shared with other checks (ex: `Kube-scheduler`)

Reply:
Agreed, very much worth moving all these helpers to the base, this one is different from what you see in `kube-scheduler` - we have `_histogram_from_microseconds_to_second` and `_summary_from_microseconds_to_second`. There's an opportunity for code reuse there as well. If you don't mind I'd prefer to introduce these changes in a separate PR, we'd also need to use a lambda to capture the metric name mapping (different across different integrations).